packages/expo-modules-autolinking/src/index.ts

```diff
@@ -36,6 +36,18 @@ function registerSearchCommand<OptionsType extends SearchOptions>(
       'ios'
     )
     .option('--silent', 'Silence resolution warnings')
+    .addOption(
+      new commander.Option(
+        '--project-root <projectRoot>',
+        'The path to the root of the project'
+      ).default(process.cwd(), 'process.cwd()')
+    )
+    .option(
+      '--only-project-deps',
+      'For a monorepo, include all modules even if they are not in the project dependencies.',
```

Comment:
Isn't it the opposite?
```suggestion
      'For a monorepo, include only modules that are the project dependencies.',
```